app/components/search_box.html.erb

```diff
@@ -9,6 +9,20 @@
           )
       %>
     </div>
+
+    <div class="search-box__input">
+      <%= f.govuk_collection_select :type,
+        [
+          OpenStruct.new(id: "", name: ""),
+          OpenStruct.new(id: "ParticipantProfile::ECT", name: "ECT"),
+          OpenStruct.new(id: "ParticipantProfile::Mentor", name: "Mentor"),
+          OpenStruct.new(id: "ParticipantProfile::NPQ", name: "NPQ")
+        ],
+        :id,
+        :name,
+        label: { text: "Filter by type" } %>
+    </div>
```

Comment:
Can we make it stay after search, like query?

Reply:
Yes. Actually this has pointed me to another bug whereby this component is shared on other views such as schools. so let me fix that too

Reply:
Ok, filters are now passed in which solves the "used in different area problems". One can now specify the value so it'll get passed down to the filter to select it.